src/pages/matches/[id].tsx

```diff
@@ -151,74 +96,57 @@ const MatchPage = ({ data, errorMessage, errorStatus }: TSsrRole<MatchDto>) => {
           })}
         >
           <Tabs
-            value={tabValue}
+            value={activeTab}
             onChange={handleTabChange}
             aria-label="teams-notes-reports-tab"
             indicatorColor="secondary"
             textColor="inherit"
             variant="fullWidth"
-            centered
           >
-            <Tab label={`${t('NOTES')} (${homeTeamNotes?.totalDocs || 0})`} />
-            <Tab
-              label={`${t('REPORTS')} (${homeTeamReports?.totalDocs || 0})`}
-            />
-            <Tab label={`${t('NOTES')} (${awayTeamNotes?.totalDocs || 0})`} />
-            <Tab
-              label={`${t('REPORTS')} (${awayTeamReports?.totalDocs || 0})`}
-            />
+            {tabs.map(({ name, type, docCount }) => (
+              <Tab
+                key={name}
+                label={`${
+                  type === 'note' ? t('NOTES') : t('REPORTS')
+                } (${docCount})`}
```

Comment:
can't types be `NOTES` and `REPORTS` to do just that?
```suggestion
                label={`${t(type)} (${docCount})`}
```